trace/Sampling.md

```diff
@@ -33,3 +37,26 @@ The OpenCensus library samples based on the following rules:
 3. If the span is a child of a local `Span` the sampling decision will be:
    * If a "span-scoped" `Sampler` is provided, use it to determine the sampling decision.
    * Else keep the sampling decision from the parent.
+
+### QPS sampler implementation details
```

Comment:
An alternative here would be to use reservoir sampling I think, should we explore that? Sounds like it's what is used to accomplish this for other systems.

Reply:
Sure, I would look into that. The advantage of this algorithm is that we update only one global atomic variable, happy to see a small pseudocode for the reservoir.